### PR TITLE
Major issues

### PR DESCRIPTION
**Pull request by kmlporto**
Renames variables, remove commented code block, and create veriables to reuse them.

### CODE REVIEW

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java [hunk 1]
```diff
@@ -286,17 +286,17 @@ private void writeTrace() throws JsonProcessingException {
                 log.error(HEIMDALL_TRACE, mapper.writeValueAsString(this));
             }
         } else {
-            String url = Objects.nonNull(this.url) ? this.url : "";
+            String urlCurrent = Objects.nonNull(this.url) ? this.url : "";
```
Comment (smrafael) [inline heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java:289]: `currentUrl` is better

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java [hunk 2]
```diff
@@ -312,22 +312,23 @@ private void writeTrace() throws JsonProcessingException {
 
 	private void printInLogger(Logger logger) throws JsonProcessingException {
 		ObjectMapper mapper = mapper();
+		String message = mapper().writeValueAsString(this);
```
Comment (smrafael) [inline heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java:315]: `mapper().writeValueAsString(this);` can not be used if else-blocks is invoked, causing a useless processing. Please, keep the previous code.

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java (hunk 1 above)
Comment (kmlporto) [reply heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java]: ok

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java (hunk 2 above)
Comment (kmlporto) [reply heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java]: ok

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java [hunk 3]
```diff
@@ -315,19 +315,19 @@ private void printInLogger(Logger logger) throws JsonProcessingException {
 
 		if (isInfo(this.resultStatus)) {
 			if (isMongo(logger))
-				logger.info(mapper.writeValueAsString(this));
+				logger.info(mapper().writeValueAsString(this));
```
[the remark this answers is not in the document]
Comment (kmlporto) [reply heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java]: ok

**Pull request merged**